yarn.lock

```diff
@@ -8836,6 +8836,11 @@ react-fast-compare@^2.0.1:
   resolved "https://registry.yarnpkg.com/react-fast-compare/-/react-fast-compare-2.0.4.tgz#e84b4d455b0fec113e0402c329352715196f81f9"
   integrity sha512-suNP+J1VU1MWFKcyt7RtjiSWUjvidmQSlqu+eHslq+342xCbGTYmC0mEhPCOHxlW0CywylOC1u2DFAT+bv4dBw==
 
+react-ga@^2.5.7:
+  version "2.5.7"
+  resolved "https://registry.yarnpkg.com/react-ga/-/react-ga-2.5.7.tgz#1c80a289004bf84f84c26d46f3a6a6513081bf2e"
+  integrity sha512-UmATFaZpEQDO96KFjB5FRLcT6hFcwaxOmAJZnjrSiFN/msTqylq9G+z5Z8TYzN/dbamDTiWf92m6MnXXJkAivQ==
+
```

Comment:
_history_ is not included in the changes, are you sure that you are including all changes of _yarn.lock_?

Reply:
mm that's weird, I'm including every change of thread. What change do you think is missing?

Reply:
Nah, maybe we already had that dependency (as a dependency of another dependency) and for that reason, it does not impact in the lock file.

Reply:
Yes, the dependency history already had.